platforms/sdl/AppPlatform_sdl.cpp

```diff
@@ -1,29 +1,16 @@
 #include "AppPlatform_sdl.hpp"
 
-#include <sstream>
-#include <fstream>
 #include <sys/stat.h>
 
-#ifndef __EMSCRIPTEN__
 #include <png.h>
 
 #include "compat/GL.hpp"
-#else
-#include <emscripten.h>
-#endif
 
 #include "client/common/Utils.hpp"
 
 AppPlatform_sdl::AppPlatform_sdl(std::string storageDir, SDL_Window *window)
+    : AppPlatform_sdlbase(storageDir, window, loadTexture("gui/default_world.png"))
```

Comment:
Don't do this. Instead, load the texture in an init function or something.

No telling what works and what doesn't during a global constructor.

(if this is actually allocated using `new AppPlatform_XXX`, let me know)

Edit: nevermind, since you allocate it with new it's probably fine.